templates/common/infra/bicep/core/host/appservice.bicep

```diff
@@ -57,7 +57,7 @@ resource appService 'Microsoft.Web/sites@2022-03-01' = {
     httpsOnly: true
   }
 
-  identity: managedIdentity ? { type: 'SystemAssigned' } : null
+  identity: { type: managedIdentity ? 'SystemAssigned' : 'None' }
```

Comment:
I think we also need to add a min version of 0.12.1 if we make this change.  The dependency check in Go.

Reply:
Will verify this.

Reply:
Confirmed this is NOT a breaking change and will NOT require users to upgrade to latest bicep version.